openfl-tutorials/interactive_api/PyTorch_Kvasir_UNet/director/director_config.yaml

```diff
@@ -1,5 +1,5 @@
 settings:
-  listen_host: localhost
+  listen_ip: localhost
```

Comment:
Are we sure that `listen_host` was changed to `listen_ip`?

Reply:
yes, otherwise, in federated mode I got `RuntimeError: Failed to bind to address <address>:50050`